### ISSUE TITLE
Issues with Webpack cause by flux-standard-action

### ISSUE THREAD

**Issue by juancarlosfarah** [acdlite/redux-promise#31]
I'm using `redux-promise` and recently I updated my application from Babel 5 to Babel 6 and I'm running into issues building with Webpack.

> ERROR in ./~/flux-standard-action/lib/index.js
Module build failed: ReferenceError: [BABEL] node_modules/flux-standard-action/lib/index.js: Unknown option: node_modules/flux-standard-action/.babelrc.stage

The error is caused by `flux-standard-action`, which in my case is only used by `redux-promise`. 

```
├─┬ redux-promise@0.5.1
│ └─┬ flux-standard-action@0.6.0
│   └─┬ lodash.isplainobject@3.2.0
│     ├── lodash._basefor@3.0.3
│     └── lodash.keysin@3.0.8
```

It seems to have been fixed ([according to this ticket](https://github.com/acdlite/flux-standard-action/pull/27)) in version 0.6.1. Can you update the dependency to 0.6.1?

**Comment by juancarlosfarah** [comment]
I see @shinnn has a pull request for this already. https://github.com/acdlite/redux-promise/pull/26

**Comment by shinnn** [comment]
@juancarlosfarah Yes, still waiting the @acdlite's review and merging.

**Comment by acdlite** [comment]
Fixed in 0.5.2
